front/src/components/CTF/Info.vue

```diff
@@ -1,47 +1,78 @@
 <template>
   <div class="background-logo" :style="style">
-    <div class="column q-px-md">
-      <div class="col q-py-md">
-        <div class="row q-gutter-md items-center">
+    <div class="column q-gutter-md">
+      <div class="col">
+        <div
+          class="row q-gutter-md items-start no-wrap"
+          style="width: calc(100vw - 16px)"
```

Comment:
I strongly dislike that the edit and delete buttons are miles away from the CTF title. Also the stacking with the weight and CTFTime link is not nice. I would suggest to move them back next to the title to make the buttons as close as possible to what they refer to.